composer.json

```diff
@@ -22,6 +22,7 @@
   "require-dev": {
     "doctrine/cache": "~1.3",
     "doctrine/orm": "~2.3",
+    "fzaninotto/faker": "^1.8",
```

Comment:
@norkunas so you suggest to remove faker here? and check if faker is installed that's it?

Reply:
No, it's all right, we need it on dev so we can test it :) You should add some tests for fakeip plugin.